### PR TITLE
Extend DNS ConfigMap tests to support IPv6

### PR DESCRIPTION
**Pull request by rajansandeep**
/kind feature

**What this PR does / why we need it**:
Extends the existing DNS ConfigMap e2e tests to include IPv6

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by MrHohn — APPROVED** [review]
Thanks, this LGTM overall.

File: test/e2e/network/dns_common.go [hunk 1]
```diff
@@ -112,14 +112,15 @@ func (t *dnsTestCommon) runDig(dnsName, target string) []string {
 		cmd = append(cmd, "@"+t.dnsPod.Status.PodIP)
 	case "kube-dns":
 		cmd = append(cmd, "@"+t.dnsPod.Status.PodIP, "-p", "10053")
+	case "ptr-record":
```
[the remark this answers is not in the document]
Comment (rajansandeep) [reply test/e2e/network/dns_common.go]: I couldn't quite follow. 
The `ptr-record` should work for both `ipv4` and `ipv6`.

**Pull request merged**